x-pack/test/idp-fixture/Vagrantfile

```diff
@@ -10,15 +10,20 @@ Vagrant.configure("2") do |config|
     config.cache.scope = :box
   end
 
-  config.vm.network "forwarded_port", guest: 389, host: 60389, protocol: "tcp"
-  config.vm.network "forwarded_port", guest: 636, host: 60636, protocol: "tcp"
```

Comment:
This one is used in x-pack/qa/openldap-tests/src/test/java/org/elasticsearch/test/OpenLdapTests.java I haven't checked the rest. I think the proper way to fix this is to move the fixture to docker like we did for the smb fixture.

Reply:
should we quit this approach then?

Reply:
I think so, we do want all fixtures to be docker based, so anything we do here would need to re-done anyhow.

Reply:
Given the discussion here, I am going to close this PR.